### ISSUE TITLE
Recovery code after log-in usage test

### ISSUE THREAD

**Issue by nowhub** [tutao/tutanota#4604]
As an operator, I want to find out whether showing users the recovery code after their first log-in is better as compared to on signup.

### Acceptance Criteria
- There should be a new task (first task should be opt-in for usage test) in the news section to configure a recovery code. (depends on #4608)
- Create a usage test that measures how users interact with the recovery code dialog after log-in (if opted in)
  - Measure interactions with the recovery code text, print button, copy to clipboard button
- ~~Recovery code is not shown anymore on final signup page~~ --> #4609
- ~~Last page for Free signup displays congratulation text~~ --> #4609

### Tasks
- [x] ...

### Depends on
- #4608 
- #4609

**Comment by nowhub** [comment]
### Test notes
- [ ] Configure the test `recoveryCodeDialog`
  - [ ] 3 Stages
  - [ ] Unlimited participations
  - [ ] 100% participating
  - [ ] Stage (index) 2: minPings: 1, maxPings: 5, enum metric `action` with allowed values: `print`, `copy`, `dismiss`, `close`, `select`
- [ ] Configure news item `usageOptIn`, disable "goes live date"
- [ ] Configure news item `recoveryCode`, disable "goes live data"
- [ ] Log in to client with user that has not opted in to usage data, confirm that both news items are being shown
- [ ] Opt in to usage data, immediately click "done" and confirm the dialog on the recovery code news item, confirm that stages are sent and that the "dismiss" action is recorded for stage 2
- [ ] Log in with another not opted-in account, opt in to usage data, immediately click "display recovery code" on the recovery code news item, enter password, select recovery code, click copy, click print, confirm that all actions are recorded for stage 2
- [ ] Log in with an opted-in account, click  "display recovery code" on the recovery code news item, enter password, do not interact with the dialog, close news, confirm that only the "close" action is logged in stage 2
